include/browseview_func.php

```diff
@@ -723,7 +717,7 @@ function getVernacularEntries($letter = "", $langcode = "", $page)
 
 	$sql = "SELECT SQL_CALC_FOUND_ROWS ID, post_content ";
     $sql .= " FROM $wpdb->posts ";
-	$sql  .= " WHERE post_content_filtered = '" . $letter ."' " . $collate . " AND post_content_filtered != '' ";
+	$sql .= " WHERE post_content_filtered = '" . $letter ."' " . $collate . " AND post_content_filtered != '' ";
```

Comment:
It looks like this could allow SQl injection.

Reply:
Wow, nice catch. Converted to use wpdb->prepare